### ISSUE TITLE
User Feedback

### ISSUE THREAD

**Issue by amilash** [raft-tech/TANF-app#1241]

**Comment by reitermb** [comment]
![image](https://user-images.githubusercontent.com/28515874/129760936-0c928c3b-193b-47fb-b587-55e29947b536.png)

**Comment by stevenino** [comment]
Can this be an email address for 2.0?

**Comment by lilybl1** [comment]
@reitermb @stevenino @lfrohlich @ADPennington To collaborate - [please use this link to view the Microsoft Forms](https://forms.office.com/Pages/DesignPageV2.aspx?subpage=design&FormId=A7fz7DcwFEeft-JMmPSZMZhIDb2i-lxLv1cIDINdQ8hUNjhUTVUyU0c5VEhJSTFJSVBVWUxCVlNFTi4u&Token=f68a5c2cfc884766a5718f4d1aa401af).

**Comment by lilybl1** [comment]
Moving this to QASP - https://reitermb.github.io/Test/knowledge-center/give-feedback.html

**Comment by lfrohlich** [comment]
FYI I got this error when I tried to open the teams link, but i can see it in the knowledge center
![image](https://user-images.githubusercontent.com/61251539/173625552-a642743b-737c-41e7-b43f-0e2b9bda86f0.png)

**Comment by lilybl1** [comment]
@lfrohlich

1. updated the questions slightly. I also broke it out into 3 specific questions. 
2. The responses are only shared with those who can collaborate - you can export the responses as well - As this form will only be used short term during Pilot Testing 2.0, the initial plan is for Miles and I to be the main recipients and viewers. As we receive responses, we will synthesize and report out during UX sync. 
3. CTA for office hours, I'll need to add that separately to the Give Feedback section because I can't embed within the Forms itself. I'll take a look at it.

**Comment by lilybl1** [comment]
@lfrohlich The CTA for office hours is at the end on the Thank you page. I didn't want to include before the survey was submitted as the office hours page will take you away from the survey and people might forget to come back to submit the survey.

**Comment by lfrohlich** [comment]
@lilybl1 that makes sense. A few edits: 

In addition to **upcoming formal rounds of user research** we'd love to offer up some unstructured time to hear from you. If you have questions about the project, feedback about elements of the current system you'd like to see TDP address, or have anything else in mind, **please sign up for an "office hours" time slot here:**

**Comment by stevenino** [comment]
LGTM!

**Comment by stevenino** [comment]
@ttran-hub

**Comment by stevenino** [comment]
@lilybl1 what would the lift be to change it everywhere else?

**Comment by lilybl1** [comment]
I'm not just referring to the Knowledge Center - I'm looking at our prod site...  https://tanfdata.acf.hhs.gov/ - my understanding is that changes here would be a new dev ticket, right?

**Comment by stevenino** [comment]
Yeah, I agree. Just trying to say that the lift would be much larger.

**Comment by ADPennington** [comment]
@lilybl1 this looks good to me. like the reframed questions based on lauren's input. not a strong preference here but the voice switch (i.e. i like....we want...) may need some formatting so response prompt is easily distinguishable from the instructions we're offering.

**Comment by lfrohlich** [comment]
@lilybl1  are you referring to message around office hours? I don't think that is in the prod site, only project updates. There's a typo in the messaging ("In addition to many upcoming structure and planned rounds") and the latter change is to make it a more direct call to action.

**Comment by lilybl1** [comment]
I pulled the content from a screenshot from Project Updates. I fixed the typo in the first sentence and added the last bit to this survey. My original question was stemming from how consistent do we want to keep this language and do Project Updates in prod need to updated as well?

**Comment by ttran-hub** [comment]
> Moving this to QASP - https://reitermb.github.io/Test/knowledge-center/give-feedback.html - Please review @lfrohlich @ADPennington @stevenino @reitermb

@ADPennington @lilybl1 @reitermb, I was tagged for A11y but no longer see it here.  If the Knowledge-center page is in the stage of modifying, please double check the findings below:
- [siteNav](https://reitermb.github.io/Test/knowledge-center/index.html) - may be a false positive, but please check the <ul class="usa-sidenav"> was flag for List element has direct children that are not allowed inside <li> elements
- Getting Started Welcome to  [TANF Data Portal (TDP)](https://reitermb.github.io/Test/knowledge-center/index.html) – First sentence, tag as subheading H4, change H4 tag to <P> or H2 
- [Creating a New Login.gov Account](https://reitermb.github.io/Test/knowledge-center/create-new-login.html)– subheading 1 to 12 are tags as H4, change tags to H2.  Since the page contains great screen sample and the page is lengthy, can we add a back-to-top navigation link at the end of the page?
- Use Existing Login.gov Account](https://reitermb.github.io/Test/knowledge-center/existing-login.html) – same comment as creating new login page
- [Submitting Data](https://reitermb.github.io/Test/knowledge-center/January-2022-Update.html) – 404 error, file not found
- [Submissions/Resubmissions 1637](javascript:void(0);) – missing link to #1637
- [Resubmitting Complete Data Files](https://reitermb.github.io/Test/knowledge-center/complete-resubmissions.html) – “Navigate to the “TANF Cases”…” subheading is tag as H4, change to H3, Since the page contains great screen sample and the page is lengthy, can we add a back-to-top navigation link at the end of the page?
- [Managing Your Account #1653](https://reitermb.github.io/Test/knowledge-center/August-2021-Update.html) – 404 error, file not found

**Comment by lfrohlich** [comment]
> I pulled the content from a screenshot from Project Updates. I fixed the typo in the first sentence and added the last bit to this survey. My original question was stemming from how consistent do we want to keep this language and do Project Updates in prod need to updated as well?

Let's discuss after standup today

**Comment by lfrohlich** [comment]
Content looks good with latest changes. No changes need to be made to project updates wording. 

@ttran-hub I moved your comments to #1693. Can you confirm if you also a11y reviewed the "Give Feedback" tab of this page?

**Comment by ttran-hub** [comment]
Hi @lfrohlich, the a11y reviewed also includes the "Give Feedback" tab

**Comment by lfrohlich** [comment]
Thanks @ttran-hub ! This is now complete
